### ISSUE TITLE
Selecting all audio tracks

### ISSUE THREAD

**Issue by briangoodman** [donmelton/other_video_transcoding#188]
The help file excerpt below leads me to believe there is no parameter that selects all audio tracks without excluding the main track.
    --add-audio TRACK|all|LANGUAGE|STRING[=WIDTH]
                    add single audio track by number
                      including main audio track
                    or all audio tracks
                      excluding main audio track

Is there a set of --add-audio parameters that will select all audio tracks?

Thanks,

Brian

**Comment by donmelton** [comment]
@briangoodman Correct. There is no option to select all audio tracks and exclude the main audio track. All is all.

Why do you want to exclude the main audio track?

**Comment by briangoodman** [comment]
Don,

I do want all tracks. Thanks for the clarification.

Regards, 

Brian

P.S. The way the help is formatted made me think that the all parameter actually excludes the main track. Take a look at the attached help file screenshot. It gives the definite impression that the main track is excluded using the All parameter.

Regards,

Brian
![Screen Shot 2023-07-10 at 6 34 28 PM](https://github.com/donmelton/other_video_transcoding/assets/1771448/a1118ced-e74c-44fa-9482-198876fab6b7)

**Comment by donmelton** [comment]
@briangoodman Yes, you're correct then in that meaning. But that's only because you can't deselect the main audio track. This behavior also allows you to set the width of all other audio tracks besides the main track.

**Comment by briangoodman** [comment]
Thanks Don. 
In that case, it might be worth amending the help file for greater clarity:
“or all audio tracks
    excluding main audio track (which cannot be deselected)”
